examples/rust/tcp-push-pop.rs

```diff
@@ -44,25 +46,18 @@ pub const SOCK_STREAM: i32 = libc::SOCK_STREAM;
 // Constants
 //======================================================================================================================
 
-const BUFFER_SIZE: usize = 64;
-const ITERATIONS: usize = 10;
+const BUFSIZE_BYTES: usize = 64;
 const FILL_CHAR: u8 = 0x65;
-const DEFAULT_TIMEOUT: Duration = Duration::from_secs(30);
-
-//======================================================================================================================
-// mksga()
-//======================================================================================================================
+const ITERATIONS: usize = 10;
+const TIMEOUT_SECONDS: Duration = Duration::from_secs(30);
```

Comment:
This might be a good time to reconsider this time out (or configure our TCP retry timeouts on the CI VMs to be lower than this).

Reply:
We discussed offline, will experiement with varying timeouts at various stages: per test timeout, test invocation timeout on cli etc. Then we will decide the right values for this.